tortoise/backends/base/schema_generator.py

```diff
@@ -327,6 +327,9 @@ def _get_m2m_tables(
             table_name = field_object.through
```

Comment:
There is no semantic difference between `db_table` and `table_name` names but actually they are very different. Let's rename `table_name` to `through_table_name`.